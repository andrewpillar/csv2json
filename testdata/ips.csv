ip_address
22.141.65.229
49.121.172.53
249.34.4.233
239.53.27.205
193.46.183.199
117.217.230.230
104.102.149.227
68.182.190.168
146.152.121.217
42.197.129.124
