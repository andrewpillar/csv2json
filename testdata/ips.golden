{"ip_address":"22.141.65.229"}
{"ip_address":"49.121.172.53"}
{"ip_address":"249.34.4.233"}
{"ip_address":"239.53.27.205"}
{"ip_address":"193.46.183.199"}
{"ip_address":"117.217.230.230"}
{"ip_address":"104.102.149.227"}
{"ip_address":"68.182.190.168"}
{"ip_address":"146.152.121.217"}
{"ip_address":"42.197.129.124"}
